### ISSUE TITLE
Regenerate controlled parquet files for train/val/test

### ISSUE THREAD

**Issue by kvantricht** [WorldCereal/presto-worldcereal#24]
Compatible parquet files for:
- Original monthly TS
- Original dekadal TS
- Presto embeddings using current best finetuned model

Parquet files should include the CatBoost label from WorldCereal for benchmarking purposes.

**Comment by kvantricht** [comment]
as discussed on slack, `test` will be kept private, `train` and `val` will be merged to allow for dynamic generation of hold-out set based on benchmarking scenarios.

**Comment by kvantricht** [comment]
One thing to keep in mind @CButsko and @gabrieltseng is that now I'm improving a lot on the consistency and comparability of the data by e.g. applying the rules to throw out suspicious ref_ids. But this means the amount of data will be reduced. For the finetuning experiments, it's better to have high-quality labels (as far as we can tell), but we need to realize that we are throwing away samples that could still be useful for pretraining for which the labels don't matter. Those samples will now not be part of the new parquet files.
